authors/apps/profiles/tests/test_profile.py

```diff
@@ -1,8 +1,7 @@
-
+from django.urls import reverse
 from rest_framework import status
 
 # local imports
-# from .test_config import TestConfiguration
 from ..models import Profile
```

Comment:
Use absolute imports here.

Reply:
If I may ask, what is the benefit of an absolute import over a relative one?

Reply:
Absolute imports are more readable than relative. 
Also, relative imports might work on one machine/server but not the other since root paths configurations are not the same for everyone.
Thus, it is always safe to specify the full path